db/migrate/20150928200909_create_pieces.rb

```diff
@@ -0,0 +1,14 @@
+class CreatePieces < ActiveRecord::Migration
+  def change
+    create_table :pieces do |t|
+
+      t.integer :row_position
+      t.integer :col_position
+      t.boolean :alive
+      t.string  :color
+      t.string  :piece_type
```

Comment:
For single table inheritance to be setup easily in rails, it's best to have the column named "type" (instead of "piece_type").